src/Database/Relations/AttachOneOrMany.php

```diff
@@ -1,303 +1,9 @@
 <?php namespace Winter\Storm\Database\Relations;
 
-use Illuminate\Database\Eloquent\Model;
-use Illuminate\Database\Eloquent\Builder;
-use Winter\Storm\Support\Facades\DbDongle;
-use Winter\Storm\Database\Attach\File as FileModel;
-use Symfony\Component\HttpFoundation\File\UploadedFile;
-
+/**
+ * @deprecated 1.2.0 Use `\Winter\Storm\Database\Relations\Concerns\AttachOneOrMany` instead.
```

Comment:
Due to the naming of the classes, it was getting a bit too hard to track what was an actual relation vs. what was a trait when all these files were mixed in the same directory under `src/Database/Relations`. I thought a tidy up would be useful here, in making sure that the traits were in a concerns subdirectory, and only the relations were kept in the main directory.

At the same time, I realised people could've been using the traits directly, so I had to keep the stubs in place. I would say though that it would be beneficial to remove them in the next main release.

TL;DR: Just clean-up of the codebase.

Reply:
How likely is it that people were actually using these traits? What would be the use case for that? If we're doing it to cleanup and it's not that likely that they were actually being used then we might as well just remove them in this version, otherwise we're not really seeing the benefits of the cleanup.

Reply:
I don't think it would be very likely anyone was using the traits directly.

Reply:
Let's just document the change and remove the original traits then.